amclient/amclient.py

```diff
@@ -459,6 +459,17 @@ def get_ingest_status(self):
             headers=self._am_auth_headers(),
         )
 
+    def get_unit_status(self, uuid):
```

Comment:
Should we call it `get_transfer_status` instead since this help is not going to work with SIPs?

Reply:
Well, there's already a function called `get_transfer_status()` (line 434), so I was following the pattern for `close_completed_transfers()`, `close_completed_ingests()` and `_close_completed_units()`.

Also...I thought SIPs were available at the `/api/transfer/` endpoint, or have I been mistaken all these years?

Reply:
I don't think so! We have a view for both types but I think [it should return a 400 error](https://github.com/artefactual/archivematica/blob/ef3af9fb58ac700dfca2a350095ef13a0c396e9c/src/dashboard/src/components/api/views.py#L247-L267).

Reply:
Maybe we should stick with get_unit_status, it kind of makes sense!

We have to find a good docstring. Something along these lines? (pls excuse my English)

> Look up the status of a transfer throughout transfer/ingest.

Reply:
how about 
```
Look up the status of an ingest or transfer unit.
```

Reply:
But is it true that you can look up the status of an ingest unit?
If you passed the ID of a SIP, wouldn't L464 raise a requests.exceptions.HTTPError?

Reply:
Right! Changed to:
>Look up the status of an ingest or transfer unit using the transfer UUID.